recipe/meta.yaml

```diff
@@ -50,15 +49,14 @@ outputs:
         - fonttools >=4.22.0
         - freetype >=2.10
         - kiwisolver >=1.0.1
-        - numpy >=1.20
         - {{ pin_compatible('numpy') }}
         - packaging >=20.0
         - pillow >=6.2.0
         # See https://github.com/matplotlib/matplotlib/pull/26158
         - pyparsing >=2.3.1,<3.1
         - python-dateutil >=2.7
         - tk >=8.6.12,<8.7  # [linux]
```

Comment:
Do we really need tk at runtime? I see that `tkinter` is used from python code, but `tkinter` is provided by CPython. Which means that the dependency to tk should come from the python package no?

Reply:
Here is an excellent conda-forge [comment](https://matplotlib.org/stable/devel/dependencies.html#backends) about the `tk` backend  https://github.com/conda-forge/matplotlib-feedstock/pull/157#issue-325602391. 
I think here we explicitly say that we use this backend instead of `pyqt`. But I would add a comment why `tk` is really mentioned

Reply:
I was wondering the same to be honest. I will remove it. conda-forge has pypy python which may not include tk. But for us we know for sure it's there. Besides, Windows and Mac also rely on tk, while we only include the dependency for linux because for some reason we only have tk as a separate package on linux.